src/rai_tts/rai_tts/tts_clients.py

```diff
@@ -62,7 +73,7 @@ def synthesize_speech_to_file(self, text: str) -> str:
                 audio_data = b"".join(response)
                 return self.save_audio_to_file(audio_data, suffix=".mp3")
             except Exception as e:
-                logger.warn(f"Error occurred during sythesizing speech: {e}.")  # type: ignore
+                logger.warn(f"Error occurred during synthesizing speech: {e}.")  # type: ignore
                 tries += 1
         audio_data = b"".join(response)
```

Comment:
If TTS will not respond withing `TTS_TRIES` this variable is unbound.

Reply:
Fixed https://github.com/RobotecAI/rai/pull/247/commits/829b91f925ffe2faf3b1f7b13dbb9a65f373da22